### PR TITLE
XS✔ ◾ Updated rule - use vertical text

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Updated as per my conversation with @adamcogan, @GordonBeeming and @calumjs 
Link to related PBI: https://github.com/SSWConsulting/SSW.Website/issues/1842

> 2. What was changed?

✏️ Nothing

> 3. Did you do pair or mob programming (list names)?

✏️ No

### CODE REVIEW

File: rules/use-vertical-text/rule.md [hunk 1]
```diff
@@ -35,12 +35,18 @@ To
 ![Figure: Bad example - Not using vertical text for headings, when you have lots of thin columns](RS_VerticalText_1.gif)  
 :::
 
-::: bad  
-![Figure: Bad example - Not using ticks and crosses](RS_VerticalText_2.gif)  
+::: good
```
Comment (tiagov8) [inline rules/use-vertical-text/rule.md:38]: Hey @jeoffreyfischer 

We have an OK class to be used here ;)

File: rules/use-vertical-text/rule.md [hunk 2]
```diff
@@ -35,12 +35,18 @@ To
...
 ::: good  
-![Figure: Good example - The report width has been reduced by clever use of the vertical text in the column headers](RS_VerticalText_3.gif)
+![Figure: Good example - Use emojis to improve the quality of your report](RS_VerticalText_3.gif)
 :::
 
+Emojis can add visual interest and make your report more engaging and easier to interpret at a glance.
+
 **Note:** Microsoft have not given us the option of having the vertical text bottom to top. It would be easier to read. This suggestion has been added to Suggestions for Microsoft RS.
+
+**Note:**
```
Comment (tiagov8) [inline rules/use-vertical-text/rule.md:52]: Hey @jeoffreyfischer 

This is an empty "Note:"

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/use-vertical-text/rule.md (hunk 1 above)
Comment (jeoffreyfischer) [reply rules/use-vertical-text/rule.md]: Done - changed "good" to "ok"

File: rules/use-vertical-text/rule.md (hunk 2 above)
Comment (jeoffreyfischer) [reply rules/use-vertical-text/rule.md]: Done - removed note

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**